nuxeo-routing-rest-api/src/test/java/org/nuxeo/ecm/restapi/server/WorkflowEndpointTest.java

```diff
@@ -659,10 +665,14 @@ public void testReassignTask() throws IOException {
         try (CloseableClientResponse response = getResponse(RequestType.GET, "/task", null, queryParams, null, null)) {
             JsonNode node = mapper.readTree(response.getEntityInputStream());
             node = node.get("entries").elements().next();
-            assertActorIs("members", node);
+            assertThatContainsActors(singletonList("members"), node.get("actors"));
         }
     }
 
+    /**
+     * @deprecated Since 10.3. Use {@link #assertThatContainsActors} instead.
+     */
+    @Deprecated
```

Comment:
Don't we still want to make assertions on a single actor in some cases?

Reply:
Yes indeed, what I meant is assertions on one and only one actor. But maybe it's not that relevant.